src/EPR.SubsidiaryBulkUpload.Application/Services/BulkSubsidiaryProcessor.cs

```diff
@@ -16,7 +16,10 @@ public class BulkSubsidiaryProcessor(ISubsidiaryService organisationService, ICo
 
     public async Task<int> Process(IEnumerable<CompaniesHouseCompany> subsidiaries, CompaniesHouseCompany parent, OrganisationResponseModel parentOrg, UserRequestModel userRequestModel)
     {
-        var subsidiariesAndOrg = subsidiaries
+        // companeis with none null company house number
+        var noneNullCompaniesHouseNumberRecords = subsidiaries.Where(ch => !string.IsNullOrEmpty(ch.companies_house_number) && string.IsNullOrEmpty(ch.franchisee_licensee_tenant));
```

Comment:
Rename to `nonNullCompaniesHouseNumberRecords` and remove the comment above, since it just says the same as the variable name

Reply:
done